rescript/plugin_setup.py

```diff
@@ -483,20 +485,44 @@
         'replacement_map': 'A tab-delimitad metadata file in which '
                            'the rank-lineage string in the \'id\' '
```

Comment:
maybe just say "string" here and below? After all, in theory this could be used to modify all sorts of information (e.g., rank labels), not just rank lineage information.